src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -143,6 +143,15 @@ impl<'a, 'tcx> TerminatorCodegenHelper<'a, 'tcx> {
             }
 
             if let Some((ret_dest, target)) = destination {
+                // Generate debug label
+                if bx.cx().tcx().sess.opts.cg.tracer.sir_labels() && bx.cx().has_debug() {
+                    let did = fx.instance.def.def_id();
+                    let lbl_name = CString::new(format!("__YK_RET_{}_{}_{}", did.krate.as_u32(),
```

Comment:
At some point, all of our labels should use a crate hash (`tcx.crate_hash(did)`) instead of a crate number. This can be done in a future PR.

The problem with crate numbers is that they change depending upon which dependency is being built, whereas crate hashes are stable over the entire compilation session.